### DISCUSSION TITLE
Default Field Label Format

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#4705] [Help]
What is the best way to change the default style for all the field labels in a form.  We want to make the 25px font?  What is the syntax to do this or other CSS changes?

**Comment by mjashanks** [comment]
Hey @CovantaDataManagement 

Goto
1. Layouts
2. Select your layouts (probably "Navigation Layout")
3. Click on "Edit Custom CSS"

enter: `--spectrum-fieldlabel-m-text-size: 25px;`

This will change the size of the label for every screen that uses this layout, in that app. You can also do this at the screen/component level, if you need more granularity.

![image](https://user-images.githubusercontent.com/3524181/156023601-8e11d659-045c-458d-b49b-7502eca0aba2.png)
